classes/class-wc-rest-connect-shipping-label-preview-controller.php

```diff
@@ -33,7 +33,7 @@ public function get( $request ) {
 		}
 
 		header( 'content-type: ' . $raw_response['headers']['content-type'] );
-		echo $raw_response['body'];
+		echo $raw_response['body']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
An alternative to ignoring could be disallowing HTML, eg: `echo wp_kses( $raw_response['body'], [] );`